src/workers/parser/frameworks.js

```diff
@@ -23,8 +23,8 @@ export function getFramework(sourceId) {
 function isReactComponent(sourceSymbols) {
   const { imports, classes, callExpressions } = sourceSymbols;
   return (
-    (importsReact(imports) || requiresReact(callExpressions)) &&
-    extendsReactComponent(classes)
+    importsReact(imports) ||
```

Comment:
no, it will not. I wasn't sure why there is the check for 'extendsReactComponent', thought maybe it is for minified files where files source code from several frameworks? but I can remove the check for class and just check an import \ require if you think it is best